crates/sdk/tests/test-client/src/module_bindings/mod.rs

```diff
@@ -861,147 +861,147 @@ impl SpacetimeModule for Module {
-            "update_unique_bool" => _reducer_callbacks.handle_event_of_type::<update_unique_bool_reducer::UpdateUniqueBoolArgs, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueBool),
-            "update_unique_i128" => _reducer_callbacks.handle_event_of_type::<update_unique_i_128_reducer::UpdateUniqueI128Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueI128),
-            "update_unique_i16" => _reducer_callbacks.handle_event_of_type::<update_unique_i_16_reducer::UpdateUniqueI16Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueI16),
-            "update_unique_i32" => _reducer_callbacks.handle_event_of_type::<update_unique_i_32_reducer::UpdateUniqueI32Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueI32),
-            "update_unique_i64" => _reducer_callbacks.handle_event_of_type::<update_unique_i_64_reducer::UpdateUniqueI64Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueI64),
-            "update_unique_i8" => _reducer_callbacks.handle_event_of_type::<update_unique_i_8_reducer::UpdateUniqueI8Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueI8),
-            "update_unique_identity" => _reducer_callbacks.handle_event_of_type::<update_unique_identity_reducer::UpdateUniqueIdentityArgs, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueIdentity),
-            "update_unique_string" => _reducer_callbacks.handle_event_of_type::<update_unique_string_reducer::UpdateUniqueStringArgs, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueString),
-            "update_unique_u128" => _reducer_callbacks.handle_event_of_type::<update_unique_u_128_reducer::UpdateUniqueU128Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueU128),
-            "update_unique_u16" => _reducer_callbacks.handle_event_of_type::<update_unique_u_16_reducer::UpdateUniqueU16Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueU16),
-            "update_unique_u32" => _reducer_callbacks.handle_event_of_type::<update_unique_u_32_reducer::UpdateUniqueU32Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueU32),
-            "update_unique_u64" => _reducer_callbacks.handle_event_of_type::<update_unique_u_64_reducer::UpdateUniqueU64Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueU64),
-            "update_unique_u8" => _reducer_callbacks.handle_event_of_type::<update_unique_u_8_reducer::UpdateUniqueU8Args, ReducerEvent>(event, _state, ReducerEvent::UpdateUniqueU8),
-            unknown => { spacetimedb_sdk::log::error!("Event on an unknown reducer: {:?}", unknown); None }
-        }
+match &function_call.reducer[..] {
+						"delete_pk_address" => _reducer_callbacks.handle_event_of_type::<delete_pk_address_reducer::DeletePkAddressArgs, ReducerEvent>(event, _state, ReducerEvent::DeletePkAddress),
```

Comment:
I don't get why this changed? Not a big deal.

Reply:
Maybe a change in `fmt`.